protocols/v2/const-sv2/src/lib.rs

```diff
@@ -16,8 +16,8 @@ pub const NOISE_FRAME_HEADER_SIZE: usize = 2;
 pub const NOISE_FRAME_HEADER_LEN_OFFSET: usize = 0;
 pub const NOISE_FRAME_MAX_SIZE: usize = u16::MAX as usize;
 
-pub const INITIATOR_EXPECTED_HANDSHAKE_MESSAGE_LENGTH: usize = 170;
-pub const RESPONDER_EXPECTED_HANDSHAKE_MESSAGE_LENGTH: usize = 32;
+pub const INITIATOR_EXPECTED_HANDSHAKE_MESSAGE_LENGTH: usize = 202;
+pub const RESPONDER_EXPECTED_HANDSHAKE_MESSAGE_LENGTH: usize = 64;
```

Comment:
It might be more readable to introduce `ELLSWIFT_KEY_SIZE` and show how the numbers here are derived, see e.g. https://github.com/Sjors/bitcoin/pull/28/files#diff-8d8617960b67f9f92cb35d0f0809976463e78e3ed8d0dffea656869fe6234334R31-R32

Reply:
you are right